fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -242,6 +239,12 @@
     @Inject
     protected UpdatePropertiesService updatePropertiesService;
 
+    @Inject
+    protected HttpIdentifierConverter httpIdentifierConverter;
```

Comment:
I'm not sure you can inject this, the reason the old one was generated via a function call is because we need to pass the current path into it. 
So if we called `translator()` in `FedoraLdp` the path is `@Path("/{path: .*}")` whereas if we call it in  `FedoraVersioning` the path is `@Path("/{path: .*}/fcr:versions")`.